### DISCUSSION TITLE
I have added `Ruff` to a large Python codebase. Is it possible to add that repo to the Who's-using-ruff section of README?

### DISCUSSION THREAD

**Discussion by Sai-Suraj-27** [astral-sh/ruff#9731] [Show and tell]
Hey, everyone. I'm a big fan of `Ruff` and I have recently made a contribution to a big python based repo **replacing flake8 with ruff** (https://github.com/unifyai/ivy/pull/27779). I'm not sure where to ask this, but is it possible to add this organization/repo (**with > 13k stars ⭐**) to the Ruff's readme? This particular section (https://github.com/astral-sh/ruff?tab=readme-ov-file#whos-using-ruff).

If you are happy to add this repo. I would be happy to create the PR adding it to Ruff's readme.
This is the repo link and it's name is **ivy** (https://github.com/unifyai/ivy)

**Comment by zanieb** [comment]
Feel free to open a pull request, thanks!

**Reply by Sai-Suraj-27** [reply]
Will do it right away. Thanks.
